pymc3/tests/test_ode.py

```diff
@@ -323,6 +325,7 @@ def system(y, t, p):
         assert trace["y0"].size > 0
         assert trace["sigma"].size > 0
 
+    @pytest.mark.xfail(reason="HalfCauchy was not yet refactored")
```

Comment:
Why "was"? Shouldn't that be "is"?

Reply:
I guess you're right 🙃
But for the sake of moving forward I'd like to not rebase/force-push again, unless something else comes up.